torchaudio/functional.py

```diff
@@ -801,3 +802,132 @@ def compute_deltas(specgram, win_length=5, mode="replicate"):
     return torch.nn.functional.conv1d(
         specgram, kernel, groups=specgram.shape[1] // specgram.shape[0]
     ) / denom
+
+
```

Comment:
To compile your function with jit, you can add the decorator `@torch.jit.script` before your function. Since `@torch.jit.script` is applied recursively, only the functions exposed to the users need to be decorated.

When rerunning the tests, there may be new compilation errors that need to be addressed. :)

Reply:
Dine!